EasyPost/_base/EasyPostClient.cs

```diff
@@ -5,27 +5,30 @@
 using System.Net.Http;
 using System.Reflection;
 using System.Threading.Tasks;
+using EasyPost.Exceptions;
 using EasyPost.Exceptions.API;
 using EasyPost.Exceptions.General;
 using EasyPost.Http;
-using EasyPost.Models.API;
 using EasyPost.Models.Shared;
 using EasyPost.Utilities;
 using RestSharp;
 
+#pragma warning disable SA1300
 namespace EasyPost._base
+#pragma warning restore SA1300
 {
     public abstract class EasyPostClient
     {
-        public readonly ClientConfiguration Configuration;
+        internal readonly ClientConfiguration Configuration;
```

Comment:
I want users to be able to pull out things like their api key or timeouts so they can troubleshoot if necessary